src/redirectPage.tsx

```diff
@@ -27,20 +27,13 @@ export default function RedirectPage (): JSX.Element {
     async function doWork (config: ConfigDb): Promise<void> {
       try {
         await setConfig(config)
-        // TODO: use channel.messageAndWaitForResponse to ensure that the config is loaded before proceeding.
-        channel.postMessage({ target: 'SW', action: 'RELOAD_CONFIG' })
+        // TODO: show spinner / disable buttons while waiting for response
```

Comment:
things happen pretty fast so i don't think we want a spinner or other things, but maybe?